redis/src/cluster_async/connections_container.rs

```diff
@@ -86,11 +85,12 @@ pub(crate) enum ConnectionType {
     PreferManagement,
 }
 
-pub(crate) struct ConnectionsMap<Connection>(pub(crate) HashMap<ArcStr, ClusterNode<Connection>>);
+pub(crate) struct ConnectionsMap<Connection>(pub(crate) DashMap<ArcStr, ClusterNode<Connection>>);
```

Comment:
Do we still need `ArcStr`? IIRC, the entire interface of `DashMap` is non-mutable, so I am not sure its needed

Reply:
Changed